MemcachedChecker/MemcachedChecker.go

```diff
@@ -0,0 +1,43 @@
+package MemcachedChecker
+
+import (
+	"errors"
+	"wait4it/model"
+
+	"github.com/bradfitz/gomemcache/memcache"
+)
+
+func (m *MemcachedConnection) BuildContext(cx model.CheckContext) {
+	m.Host = cx.Host
+	m.Port = cx.Port
+}
+
+func (m *MemcachedConnection) Validate() (bool, error) {
+	if len(m.Host) == 0 {
+		return false, errors.New("Host can't be empty")
+	}
+
+	if m.Port < 0 || m.Port > 65535 {
```

Comment:
`m.Port < 1` is the correct way, kindly update it.

Reply:
I've been confused. You did this port checking in other checkers with different approaches and hence I just decided to choose one of them.